ext/MadIPMMathOptInterfaceExt/MadIPMMathOptInterfaceExt.jl

```diff
@@ -25,7 +25,7 @@ include("parse_moi.jl")
 include("MOI_wrapper.jl")
 
 function __init__()
-    setglobal!(MadIPM, :Optimizer, Optimizer)
+    @eval MadIPM const Optimizer = $Optimizer
```

Comment:
@frapac @sshin23 
Why did you change that?
It is required if we use one day `PrecompiledTools.jl` (to the best of my knowledge).
It is what we did for `Ipopt.jl` and `KNITRO.jl` with Oscar.